tests/components/logger/test_init.py

```diff
@@ -1,143 +1,190 @@
+    assert logging.getLogger(HASS_NS).isEnabledFor(logging.WARNING) is True
 
-    assert logging.getLogger("").isEnabledFor(logging.DEBUG) is True
+    assert logging.getLogger(COMPONENTS_NS).isEnabledFor(logging.DEBUG) is False
+    assert logging.getLogger(COMPONENTS_NS).isEnabledFor(logging.WARNING) is True
+    assert logging.getLogger(COMPONENTS_NS).isEnabledFor(logging.INFO) is True
 
+    assert logging.getLogger(GROUP_NS).isEnabledFor(logging.DEBUG) is False
+    assert logging.getLogger(GROUP_NS).isEnabledFor(logging.WARNING) is True
+    assert logging.getLogger(GROUP_NS).isEnabledFor(logging.INFO) is True
 
-async def test_set_filter(hass):
-    """Test change log level of existing filter."""
-    log_filter = await async_setup_logger(hass, TEST_CONFIG)
+    assert logging.getLogger(f"{GROUP_NS}.any").isEnabledFor(logging.DEBUG) is False
+    assert logging.getLogger(f"{GROUP_NS}.any").isEnabledFor(logging.WARNING) is True
+    assert logging.getLogger(f"{GROUP_NS}.any").isEnabledFor(logging.INFO) is True
 
-    assert not log_filter.filter(RECORD("asdf", logging.DEBUG))
-    assert log_filter.filter(RECORD("dummy", logging.WARNING))
+    assert logging.getLogger(ZONE_NS).isEnabledFor(logging.DEBUG) is True
+    assert logging.getLogger(f"{ZONE_NS}.any").isEnabledFor(logging.DEBUG) is True
 
     await hass.services.async_call(
-        logger.DOMAIN, "set_level", {"asdf": "debug", "dummy": "info"}
+        logger.DOMAIN, "set_level", {f"{UNCONFIG_NS}.any": "debug"}, blocking=True
     )
-    await hass.async_block_till_done()
-
-    assert log_filter.filter(RECORD("asdf", logging.DEBUG))
-    assert log_filter.filter(RECORD("dummy", logging.WARNING))
 
-
-async def test_set_default_filter_empty_config(hass):
-    """Test change default log level from empty configuration."""
-    log_filter = await async_setup_logger(hass, NO_DEFAULT_CONFIG)
-
-    assert logging.getLogger("").isEnabledFor(logging.DEBUG) is True
-    assert logging.getLogger("").isEnabledFor(logging.WARNING) is True
-
-    assert log_filter.filter(RECORD("test", logging.DEBUG))
+    logging.getLogger(UNCONFIG_NS).level == logging.NOTSET
+    logging.getLogger(f"{UNCONFIG_NS}.any").level == logging.DEBUG
+    logging.getLogger(UNCONFIG_NS).level == logging.NOTSET
 
     await hass.services.async_call(
-        logger.DOMAIN, "set_default_level", {"level": "warning"}
+        logger.DOMAIN, "set_default_level", {"level": "debug"}, blocking=True
     )
-    await hass.async_block_till_done()
-
-    assert not log_filter.filter(RECORD("test", logging.DEBUG))
-
-    assert logging.getLogger("").isEnabledFor(logging.DEBUG) is False
-    assert logging.getLogger("").isEnabledFor(logging.INFO) is False
-    assert logging.getLogger("").isEnabledFor(logging.WARNING) is True
 
+    assert logging.getLogger(UNCONFIG_NS).isEnabledFor(logging.DEBUG) is True
+    assert logging.getLogger(f"{UNCONFIG_NS}.any").isEnabledFor(logging.DEBUG) is True
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)